ext/ffi/ffi.stub.php

```diff
@@ -55,7 +55,7 @@ public static function memcpy(FFI\CData $dst, $src, int $size): void {}
     public static function memcmp($ptr1, $ptr2, int $size): ?int {}
 
     /** @prefer-ref $ptr */
-    public static function memset(FFI\CData $ptr, int $ch, int $size): void {}
+    public static function memset(FFI\CData $ptr, int $byte, int $size): void {}
 
     /** @prefer-ref $ptr */
     public static function string(FFI\CData $ptr, ?int $size = null): ?string {}
```

Comment:
ah, the second parameter is tricky! Normally, we'd use `$length`, but here we're talking about a certain `$size` of memory area. Ok, makes sense :)

Reply:
Yes, I think `$length` would be fine here as well, but as the rest of this API uses `$size`, I stuck with that.